core/utils.py

```diff
@@ -8,6 +8,12 @@
 import yaml
 
 
+def check_if_same(items: List):
```

Comment:
Useful! Could be implemented in triangulation_calibration_module.py to validate that the metadata in all recording/calibration videos is the same (target_fps, recording_date, calibration_index, etc.)